package.json

```diff
@@ -9,6 +9,7 @@
     "build": "node build/build.js",
     "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
     "e2e": "wdio wdio.conf.js",
+    "i18n": "I18N_DEBUG=true yarn e2e-with-mocks",
```

Comment:
Do we want `I18N_DEBUG` on by default? I guess we probably do?

Reply:
It's spammy and very.. hacky right now. I'd prefer us to have a look whenever necssary rather than by default
The best solution would prob be using it as a plugin/reporter, which would allow us to format it better than now.

Reply:
So are you saying you'd rather turn it off? Is it noisy when there _isn't_ missing strings. If not, then the noise is good IMO. Makes sure something is done about the missing strings.

Reply:
Well, it doesn't format the output properly because wdio outputs at every end of spec file, instead at every end of test "run" : that's because the memory is lost in-between each spec file. That means you have to scroll back for every spec file running to find the corresponding missing translations : that's crap.
That's why a reporter/plugin is probably needed, and deactivated by default : only show them when you're working & testing a single spec file :)